code/controlconfig/controlsconfig.cpp

```diff
@@ -2653,6 +2673,7 @@ void scale_invert(const CC_bind &bind,
 	}
 }
 
+int axes_last_value[Action::NUM_VALUES];
```

Comment:
int used is actively used elsewhere, hence I'd be a bit scared of that.
I specifically put it here, since it is only ever required to store the last value of an axis. Rather, only the absolute axes would suffice (as locking those keeps them at their last recorded value), but that's hard to address.
I could add another field in Config_item, but as this is very much not a "part" of the config, only applies to axes, and only a low level value cache, I thought it'd be better to keep it as a separate array.